### PR TITLE
Timeline: Sep 29 Picnic

### PR DESCRIPTION
**Pull request by dasanchez**
Adding the mesh picnic to the Timeline page.

### CODE REVIEW

**Comment by dasanchez** [comment]
I will summarize and close issue #230 with a separate file in the [documents](https://github.com/tomeshnet/documents) repo.

**Review by makew0rld — APPROVED** [review]

File: _timeline/2018-09-29-mesh-picnic.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Mesh Picnic
+date: 2018-09-29
+---
+[Free Geek Toronto](https://www.freegeektoronto.org/) hosted a technology picnic open to the public.
```
Comment (garrying) [inline _timeline/2018-09-29-mesh-picnic.md]: The double-space at the end of this line intentional? The line-break seems off to me:
screen shot 2018-10-09 at 9 48 10 pm

Comment (benhylau) [reply _timeline/2018-09-29-mesh-picnic.md]: Timeline should be one blob of text like the rest, this looks off.

**Review by darkdrgn2k — APPROVED** [review]

File: _timeline/2018-09-29-mesh-picnic.md (hunk 1 above)
Comment (dasanchez) [reply _timeline/2018-09-29-mesh-picnic.md]: Thanks @garrying @benhylau - it's one blob of text now.

**Review by garrying — APPROVED** [review]

**Review by benhylau — APPROVED** [review]

**Pull request merged**